### PR TITLE
add side nav documentation

### PR DESCRIPTION
**Pull request by carodew**
Note: Does not include the "Track progress here." sentence in the Intro text as I didn't have the link for "here." This can be merged now and we can add that later.

### CODE REVIEW

File: _components/sidenav.md [hunk 1]
```diff
@@ -117,30 +117,27 @@ lead: Intro text on what is included in this section and how to use it. No more
     <h4 class="usa-heading">Accessibility</h4>
     <p>As you customize this form template, ensure it continues to follow the:</p>
     <ul class="usa-content-list">
-      <li>Item 1</li>
-      <li>Item 2</li>
-      <li>Item 3</li>
+      <li>Ensure the side navigational system is keyboard accessible. Users should be able to tab through each link.
+</li>
```
Comment (maya) [inline _components/sidenav.md]: @carodew please put the closing `</li>` on the same line as the list.

File: _components/sidenav.md [hunk 2]
```diff
@@ -117,30 +117,27 @@ lead: Intro text on what is included in this section and how to use it. No more
...
     </ul>
     <h5>Guidelines</h5>
     <ul class="usa-content-list">
-      <li>Item 1</li>
-      <li>Item 2</li>
-      <li>Item 3</li>
+      <li>Indicate where a user is within the navigational hierarchy. Use the “active” state to show users which page they have navigated to.</li>
+      <li>Keep the navigation links short. They can be shorter derivatives of page titles themselves.</li>
+      <li>If the navigation hierarchy is too long, users may miss items at the bottom. If it’s too deep, users may miss items that require too many clicks. Usability test to find the right balance between breadth and depth.
+</li>
```
Comment (maya) [inline _components/sidenav.md]: @carodew same as https://github.com/18F/web-design-standards/pull/521/files#r39451769

File: _components/sidenav.md (hunk 1 above)
Comment (carodew) [inline _components/sidenav.md]: fixed in 968122c

File: _components/sidenav.md (hunk 2 above)
Comment (carodew) [inline _components/sidenav.md]: fixed in 968122c

**Pull request merged**